doc/cephfs/nfs.rst

```diff
@@ -8,57 +8,58 @@ CephFS namespaces can be exported over NFS protocol using the
 ========================================
 
-NFS-Ganesha provides a File System Abstraction Layer (FSAL) to plug in different
-storage backends. `FSAL_CEPH <https://github.com/nfs-ganesha/nfs-ganesha/tree/next/src/FSAL/FSAL_CEPH>`_
-is the plugin FSAL for CephFS. For each NFS-Ganesha export, FSAL_CEPH uses a
-libcephfs client, user-space CephFS client, to mount the CephFS path that
-NFS-Ganesha exports.
+NFS-Ganesha provides a File System Abstraction Layer (FSAL) to plug in
+different storage backends. FSAL_CEPH_ is the plugin FSAL for CephFS. For
+each NFS-Ganesha export, FSAL_CEPH_ uses a libcephfs client, user-space
+CephFS client, to mount the CephFS path that NFS-Ganesha exports.
 
-Setting up NFS-Ganesha with CephFS, involves setting up NFS-Ganesha's
-configuration file, and also setting up a Ceph configuration file and cephx
-access credentials for the Ceph clients created by NFS-Ganesha to access
-CephFS.
+Setting up NFS-Ganesha with CephFS, involves setting up NFS-Ganesha's and
+Ceph's configuration file and CephX access credentials for the Ceph clients
+created by NFS-Ganesha to access CephFS.
 
 NFS-Ganesha configuration
 -------------------------
 
-A sample ganesha.conf configured with FSAL_CEPH can be found here,
+A sample ``ganesha.conf`` configured with FSAL_CEPH_ can be found here,
 `<https://github.com/nfs-ganesha/nfs-ganesha/blob/next/src/config_samples/ceph.conf>`_.
 It is suitable for a standalone NFS-Ganesha server, or an active/passive
-configuration of NFS-Ganesha servers managed by some sort of clustering
+configuration of NFS-Ganesha servers, to be managed by some sort of clustering
 software (e.g., Pacemaker). Important details about the options are
 added as comments in the sample conf. There are options to do the following:
 
 - minimize Ganesha caching wherever possible since the libcephfs clients
-  (of FSAL_CEPH) also cache aggressively
+  (of FSAL_CEPH_) also cache aggressively
 
 - read from Ganesha config files stored in RADOS objects
 
 - store client recovery data in RADOS OMAP key-value interface
 
 - mandate NFSv4.1+ access
 
-- enable read delegations (need at least v13.0.1 'libcephfs2' package
-  and v2.6.0 stable 'nfs-ganesha' and 'nfs-ganesha-ceph' packages)
+- enable read delegations (need at least v13.0.1 ``libcephfs2`` package
+  and v2.6.0 stable ``nfs-ganesha`` and ``nfs-ganesha-ceph`` packages)
 
 Configuration for libcephfs clients
 -----------------------------------
 
-Required ceph.conf for libcephfs clients includes:
-
-* a [client] section with ``mon_host`` option set to let the clients connect
-  to the Ceph cluster's monitors, usually generated via ``ceph config generate-minimal-conf``, e.g., ::
+``ceph.conf`` for libcephfs clients includes a ``[client]`` section with
+``mon_host`` option set to let the clients connect to the Ceph cluster's
+monitors, usually generated via ``ceph config generate-minimal-conf``.
+For example::
 
-    [global]
+    [client]
```

Comment:
I might be misled here by the doc. Changed it to ``[client]`` because the text above says "``[client]`` section with mon_host option". Is it actually a client section _and_ a global section with mon_host option?